### ISSUE TITLE
Crash when using Live Coding with Melee Trace in C++

### ISSUE THREAD

**Issue by E3245** [rlewicki/MeleeTrace#13]
I keep getting this crash when I try to compile via live coding with MeleeTrace as a dependency. I'm not even calling anything in my Weapon constructor class, maybe except for calling ```	UPROPERTY(EditAnywhere, Category = "Gameplay") FMeleeTraceInfo WeaponTraceInfo;``` in the .h file

```
Fatal error: [File:D:\build\++UE5\Sync\Engine\Source\Runtime\CoreUObject\Private\UObject\Obj.cpp] [Line: 148] We are currently retrieving VTable ptr. Please use FVTableHelper constructor instead.

UnrealEditor_CoreUObject
UnrealEditor_MeleeTrace!InternalConstructor<UMeleeTraceShape_Line>() [D:\RocketSync\5.3.0-27405482+++UE5+Release-5.3\Working\Engine\Source\Runtime\CoreUObject\Public\UObject\Class.h:3597]
UnrealEditor_CoreUObject
UnrealEditor_MeleeTrace!UMeleeTraceShape_Line::MakeLineShape() [D:\build\U5M-Marketplace\Sync\LocalBuilds\PluginTemp\HostProject\Plugins\MeleeTrace\Source\MeleeTrace\Private\MeleeTraceShape.cpp:23]
UnrealEditor_MeleeTrace!FMeleeTraceInfo::FMeleeTraceInfo() [D:\build\U5M-Marketplace\Sync\LocalBuilds\PluginTemp\HostProject\Plugins\MeleeTrace\Source\MeleeTrace\Private\MeleeTraceInfo.cpp:10]
UnrealEditor_MyGameProject_patch_0!AWeapon::AWeapon() [D:\Big Boss\Documents\Unreal Projects\MyGameProject\Intermediate\Build\Win64\UnrealEditor\Inc\MyGameProject\UHT\Weapon.gen.cpp:540]
UnrealEditor_CoreUObject
UnrealEditor_CoreUObject
UnrealEditor_MyGameProject_patch_0!AWeapon::GetPrivateStaticClass() [D:\Big Boss\Documents\Unreal Projects\MyGameProject\Intermediate\Build\Win64\UnrealEditor\Inc\MyGameProject\UHT\Weapon.gen.cpp:334]
UnrealEditor_CoreUObject
UnrealEditor_MyGameProject_patch_0!APlayerWeapon::GetPrivateStaticClass() [D:\Big Boss\Documents\Unreal Projects\MyGameProject\Intermediate\Build\Win64\UnrealEditor\Inc\MyGameProject\UHT\PlayerWeapon.gen.cpp:21]
UnrealEditor_CoreUObject
UnrealEditor_CoreUObject
UnrealEditor_LiveCoding
UnrealEditor_LiveCoding
UnrealEditor_LiveCoding
UnrealEditor
UnrealEditor
UnrealEditor
UnrealEditor
UnrealEditor
UnrealEditor
UnrealEditor
kernel32
ntdll
```

**Comment by rlewicki** [comment]
@E3245 thanks for reaching out! That's is interesting, I think I've used live coding without any issues on version 5.3 which based on the crash log you also use. I will have a closer look to see if I can find something soon!

**Comment by rlewicki** [comment]
Hey, just wanted to let  you know that I've been able to reproduce it and am trying to see what is causing it and to patch it. Will update you once I get a resolution on this!

**Comment by rlewicki** [comment]
@E3245 I'm happy to say that I was able to fix this crash. If you urgently need this in your project then feel free to get latest version from the master branch. I will be preparing few more improvements and will try to make a new release to the marketplace and GitHub releases before the end of the week.

Thanks again for reporting the issue!

**Comment by rlewicki** [comment]
Closing the issue as this is fixed in release v1.4.0

Thanks for reporting the issue!
